### PR TITLE
Enhance installer script to setup yb_migrate on mac.

### PR DESCRIPTION
**Pull request by amit-yb**
This script doesn't install ora2pg yet. Supports only PG.

### CODE REVIEW

File: installer_scripts/install-yb-voyager [hunk 1]
```diff
@@ -99,6 +115,14 @@ ubuntu_main() {
 }
 
 
+macos_main() {
+	macos_install_brew
```
Comment (sanyamsinghal) [inline installer_scripts/install-yb-voyager:119]: can we print a Note to the user saying that its only for postgres

**Review by sanyamsinghal — APPROVED** [review]
LGTM, left few comments please respond to them before merging.

File: installer_scripts/install-yb-voyager (hunk 1 above)
Comment (amit-yb) [reply installer_scripts/install-yb-voyager:123]: Done.

File: installer_scripts/install-yb-voyager [hunk 2]
```diff
@@ -213,22 +247,30 @@ install_ora2pg() {
 
 
 update_bashrc() {
+	os=$(uname -s)
+
+	rc_file_name="${HOME}/.bashrc"
+	if [ ${os} == "Darwin" ]
```
[the remark this answers is not in the document]
Comment (amit-yb) [reply installer_scripts/install-yb-voyager]: Done.

```
update_bashrc() {
        # Either ${HOME}/.bashrc or ${HOME}/.zshrc .
        rc_file_name="${HOME}/.$(basename $(echo $SHELL))rc"
```

**Pull request merged**